docs/gaia/gaia.rst

```diff
@@ -92,15 +92,47 @@ degrees around an specific point in RA/Dec coordinates.
+                  ...                 ... ...                                                                                 ...                  ...
+  0.01687730304857345 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066940129962368                     
+  0.01697735455230481 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089548838418048                     
+  0.01723374116453639 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066115496849152                     
+  0.01737196890379023 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089578900242432                     
+  0.01747850362437053 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090369174230656                     
+ 0.017605525662679422 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089613259710208                     
+  0.01824049579793165 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090437893440640                     
+  0.01916481512715311 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066935832392960                     
+  0.01943176697471851 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089510180765312                     
+ 0.019464719601172412 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066871411763712                     
+ 0.019467068628703368 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089514475519232                     
+ 0.019752561500226976 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090407832546944                     
+  0.01991656886177004 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066940132132352                     
+ 0.020149589233310516 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066871411763968                     
+ 0.020307185970548904 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636089514478069888                     
+ 0.020454730686780127 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636066940131244288                     
+ 0.020802655215768254 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636067141990822272                     
+ 0.021615117161838747 1635721458409799680 ... https://gea.esac.esa.int/data-server/datalink/links?ID=Gaia+DR2+6636090369173963776                     
+Length = 50 rows
+
```

Comment:
I am unclear on what these changes in the Gaia docs have to do with making in code to-dos into issues. Can you clarify?

Reply:
oops 😬. I was working on this #1217, I must have mixed up the branches. Sorry, I'm still learning how to use git.
I will delete the commit.